contracts/avalanche/connectors/aave/v3-import/events.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity ^0.8.6;
+pragma experimental ABIEncoderV2;
+
+contract Events {
+	event LogAaveV3Import(
```

Comment:
@bhavik-m Can you add fee amounts over here?

Reply:
Sure.